phoenix_backend/.envrc

```diff
@@ -0,0 +1,2 @@
+source_up
```

Comment:
This is probably a useful convention but it makes me a little nervous security-wise because [sourced scripts aren't checked for security issues](https://direnv.net/man/direnv-stdlib.1.html#codesourceup-ltfilenamegtcode) and we aren't actually even making use of that functionality for this project (unless you're somehow depending on it to run `.envrc` files that are in your home directory or something, which I think is out of scope for this example).

Reply:
I was worried at first, too, until I realised the reason `dotenv` behaves that way is because the `.envrc` in the parent directory is outside its threat boundary: `.envrc` files you got by switching branches or changing directory into a fresh clone.

I didn't make the change idly: I depend on my parent `.envrc` for correct behaviour, _eg._ to make sure `asdf install` gets a working Erlang despite macOS `openssl`. If you're not at peace with the explanation above, perhaps we could move `.envrc` to `.envrc-example`?

Reply:
I feel like it’s probably fine, just wanted to point it out in case there was a big concern. If someone is using `direnv` then they will have to allow anyway, so if they’re uncomfortable they could comment out that line.

Alternately, we could remove the line from this project (since we don’t need it specifically for this project) and you could maintain it as an uncommitted local change to get your setup to work.